assignments/assignment01.rb

```diff
@@ -68,7 +99,17 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  # your implementation here
+  #make all the letters lower case
+  original = s.downcase
+
+  #remove all non alphanumeric characters
+  original.gsub!(/[^\p{Alnum}-]/, '')
+
+  #save a reverse copy
+  reverse = original.reverse
+
+  #compare and return results
+  return reverse == original ? true : false 
```

Comment:
idiomatic ruby would be to write this line simply as "reverse == original". That will generate the true/false boolean value you want, and since it's the last line in the method it will automatically be the method's return value.